var.tf

```diff
@@ -0,0 +1,128 @@
+variable "environment" {
+  description = "Environment name (dev or prod) - Set in Terraform Cloud workspace"
```

Comment:
You can remove the "Set in Terraform Cloud workspace" from each description. It's actually true of all variables, and isn't worth mentioning. (Side note, it's called HCP Terraform now, not Terraform Cloud.)